golang/Dockerfile

```diff
@@ -10,6 +10,6 @@ COPY go.mod go.sum ./
 RUN go mod download && go mod verify
 
 COPY . .
-RUN go build -v -o /usr/local/bin/app ./...
+RUN go build -v -o /usr/local/bin/app/ ./...
```

Comment:
option `-o` creates an executable(?) with name specified after `-o`. However, for multi package builds, this approach doesn't work. So, build name is changed to a path, a directory, in which executables will be created. Names of the executables will be package names.